### PR TITLE
Created Home Page

### PR DESCRIPTION
**Pull request by khyatibaria**
### Created Home Page
Please review https://deploy-preview-6--bookish-club.netlify.app/home

### CODE REVIEW

**Comment by parthpandyappp** [comment]
Added some comments, rest LGTM :+1:

File: Home/Home.css [hunk 1]
```diff
@@ -0,0 +1,115 @@
+body {
+  background-color: #f5f5f5;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
```
Comment (kotesh-arya) [inline Home/Home.css:7]: Instead of this short-hand value of "repeat(4,1fr)" will be more preferrable I think

**Review by kotesh-arya — APPROVED** [review]
Just one suggestion , rest of the code is perfect enough for a merge 👍🏻

**Review by rishimish — APPROVED** [review]
looks good to merge

**Pull request merged**